src/show/pages/public/Standings/index.jsx

```diff
@@ -2,160 +2,42 @@ import {
-  },
-
-  {
-    id: '2',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '3',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '4',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '5',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '6',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '7',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '8',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '9',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '10',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '11',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  }
-];
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 
 const Standings = () => {
+  const [constructor, setConstructor] = useState();
+
+  useEffect(() => {
+    axios
+      .get('http://localhost:3000/v1/constructors/')
+      .then(res => {
+        console.log(res.data);
```

Comment:
remove console.log

Reply:
After you remove the console log you can make this in one line.